sql/rowexec/update.go

```diff
@@ -256,9 +255,22 @@ func (u *updateJoinIter) Next(ctx *sql.Context) (sql.Row, error) {
 	}
 }
 
+func toJoinNode(node sql.Node) *plan.JoinNode {
```

Comment:
Could use `switch n := node.(type)` instead of having to recast every time
Example:
```go
func toJoinNode(node sql.Node) *plan.JoinNode {
	switch n := node.(type) {
	case *plan.JoinNode:
		return n
	case *plan.Filter:
		return toJoinNode(n.Child)
	case *plan.Project:
		return toJoinNode(n.Child)
	default:
		return nil
	}
}
```

Reply:
Yup, this is what I've changed it to with all the other node types mentioned above added. I will hopefully be able to push the updates sometime this week.